src/containers/Tablets/Tablets.tsx

```diff
@@ -71,6 +71,7 @@ function getColumns({database}: {database?: string}) {
                         additionalControls={
                             <DeveloperUILinkButton
                                 href={createTabletDeveloperUIHref(row.TabletId)}
+                                size="xs"
```

Comment:
You should decrease `TabletId` column width, it's contains unneeded empty space now (I guess 190 will be just fine)